mkvcore/interceptor.go

```diff
@@ -25,6 +26,13 @@ type BlockInterceptor interface {
 	Intercept(r []BlockReader, w []BlockWriter)
 }
 
+func MustBlockInterceptor(interceptor BlockInterceptor, err error) BlockInterceptor {
```

Comment:
Could you add a godoc comment for this?

Reply:
@at-wat Done both, okay to merge?

Reply:
Godoc comment of the func should be started by the function name.
```
mkvcore/interceptor.go:29:1: comment on exported function MustBlockInterceptor should be of the form "MustBlockInterceptor ..."
```

(I just noticed that the lint result report on CI is not working.)

Reply:
@at-wat pushed fix, didn't see that link message